fog/ledger/server/src/key_image_service.rs

```diff
@@ -73,7 +73,36 @@ impl<L: Ledger + Clone, E: LedgerEnclaveProxy> KeyImageService<L, E> {
         self.db_poll_shared_state.clone()
     }
 
-    pub fn auth_impl(
+    pub fn auth_store(
+        &mut self,
+        mut req: AuthMessage,
+        logger: &Logger,
+    ) -> Result<attest::AuthMessage, RpcStatus> {
+        // TODO: Use the prost message directly, once available
+        match self.enclave.frontend_accept(req.take_data().into()) {
```

Comment:
The difference here is that we use `frontend_accept` for the store and `client_accept` for the old-style service. In theory we eventually delete the old-style service.

Reply:
Do you plan on deleting the old-style service as well? Because isn't it true that this service won't ever be contacted by clients anymore? That is, only the router will contact it? if yes, I'd recommend making a follow up issue to get rid of this functionality to make it clear that this is the case + sync with what fog view store does.

Reply:
Yes, we do intend to delete the old-style service. I'll file the issue.